lib/uncruft/deprecation_handler.rb

```diff
@@ -41,9 +41,11 @@ def normalize_message(message)
 
     def normalize_callstack_path(message)
       if (gem_home = gem_home(message)).present?
-        message.gsub(gem_home, '$GEM_PATH')
-      elsif (absolute_path = absolute_path(message)).present?
-        message.gsub(absolute_path, relative_path(absolute_path))
+        message.gsub!(gem_home, '$GEM_PATH')
+      end
+
+      if (absolute_path = absolute_path(message)).present?
+        message.gsub!(absolute_path, relative_path(absolute_path))
```

Comment:
in the case of the ruby deprecation warning i was receiving, we were only doing one or the other, but this one had both, so adjusting this logic a little bit:

```
"DEPRECATION WARNING: PG::Coder.new(hash) is deprecated. Please use keyword arguments instead! Called from /Users/devinburnette/.rbenv/versions/3.0.5/lib/ruby/gems/3.0.0/gems/activerecord-6.1.7.3/lib/active_record/connection_adapters/postgresql_adapter.rb:883:in `new' (called from <top (required)> at /Users/devinburnette/src/coach/coach/spec/rails_helper.rb:32)"
```